### PR TITLE
Bugfix/gallery

### PR DESCRIPTION
**Pull request by Mortadha-abderrhim**
Fixed Bugs related to the Gallery.
  - When an image is clicked it is shown on full screen.
  - No weird behavior when the number of images in the gallery is less than the number of shown pages
 The low coverage is basically due to the image loader
galleryWithLessThan4
Screenshot_1617701428

### CODE REVIEW

File: app/src/main/java/com/pindex/main/ui/fragments/blocks/GalleryBlockFragment.kt [hunk 1]
```diff
@@ -24,14 +27,15 @@ class GalleryBlockFragment : Fragment(R.layout.fragment_pindex_block_gallery) {
     lateinit var viewPager: ViewPager
     private var selectedPosition: Int = 0
     lateinit var galleryPagerAdapter: GalleryPagerAdapter
+    var calling: FragmentActivity? = null
```
Comment (ToufiPF) [inline app/src/main/java/com/pindex/main/ui/fragments/blocks/GalleryBlockFragment.kt:30]: Shouldn't this rather be `lateinit var calling: FragmentActivity` ?

Comment (Mortadha-abderrhim) [reply app/src/main/java/com/pindex/main/ui/fragments/blocks/GalleryBlockFragment.kt]: Yes, fixed in the new commit.

**Review by ToufiPF — APPROVED** [review]
Looks good to me, GJ!

**Review by amomo25 — APPROVED** [review]
I don't know if it was possible to create a full screen image without reloading it entirely, but other than that the code looks great.

**Pull request merged**